src/utils/web3-utils.js

```diff
@@ -0,0 +1,50 @@
+const { isNull } = require('lodash');
+const web3 = require('../web3');
+const TransactionReceipt = require('../models/tx-receipt');
+
+/**
+ * Gets the pending tx and conforms it to the TransactionReceipt model.
+ * @param {string} txid Transaction ID
+ * @return {TransactionReceipt} Transaction receipt
+ */
+const getTransaction = async (txid) => {
+  const tx = web3().eth.getTransaction(txid);
+  return new TransactionReceipt({
+    transactionHash: tx.hash,
+    blockHash: tx.blockHash,
+    blockNumber: tx.blockNumber,
```

Comment:
tx.blockNumber is null when i get it
TransactionReceipt validation at this part has problem. see the tx returned from `web3().eth.getTransaction(txid);` 
![image](https://user-images.githubusercontent.com/10339131/57892908-b0e4fa80-77f5-11e9-83fe-80da213b968b.png)

Reply:
fixed